COMMERCE-10762 Fix query condition

smottal opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

katienesterovich commented:
ci:test:sf

katienesterovich commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-7.liferay.com/job/test-portal-source-format/1638/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-7.liferay.com//userContent/jobs/test-portal-source-format/builds/1638/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest(master)/5452/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-5.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5452/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 2 minutes 14 seconds 743 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/22cde9ad7563e8696feb1e27b1c1c6367098ddfb">22cde9ad7563e8696feb1e27b1c1c6367098ddfb</a></p>0 out of 1 jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest(master)/5452/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-5.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5452/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest(master)/5452/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre><code>A CI failure has occurred. QA Engineering has been notified. Please try again later or contact QA Engineering for an update.</code></pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes 56 seconds 513 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/22cde9ad7563e8696feb1e27b1c1c6367098ddfb">22cde9ad7563e8696feb1e27b1c1c6367098ddfb</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-103347">LPS-103347</a><br/>Branch GIT ID: <a href="https://github.com/katienesterovich/liferay-portal/commit/f364359bdea79009eaa321bc87b01d6f719b0418">f364359bdea79009eaa321bc87b01d6f719b0418</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-source-format/1638/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-source-format/builds/1638/jenkins-report.html">here</a>.</h5></details></html>

brianikim commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest(master)/5453/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-5.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5453/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 17 out of 19 jobs passed in 1 hour 20 minutes 42 seconds 851 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/22cde9ad7563e8696feb1e27b1c1c6367098ddfb">22cde9ad7563e8696feb1e27b1c1c6367098ddfb</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/925aa9971fef1c3c1f0670782c18977627b3c4f1">925aa9971fef1c3c1f0670782c18977627b3c4f1</a></p>17 out of 19 jobs PASSED<div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest(master)/5453/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200865/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</strong></strike></a></li></ul></div><details><summary><strong>17 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275858/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200864/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200866/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275860/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342981/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342980/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275855/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275856/">test-portal-acceptance-pullrequest-batch(master)/js-unit-private-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200861/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275859/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342982/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275857/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200860/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200862/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200867/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200868/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200863/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-5.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5453/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">acceptance upstream results</a> at 22cde9ad7563e8696feb1e27b1c1c6367098ddfb:</strong></summary><ol><li><div><h5><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200865/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></h5><div><h6>Job Results:</h6><p>4 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/200865//consoleText">AXIS_VARIABLE=0,label_exp=!master #200865</a><ol><li><div><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/200865//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a></div></li></ol></div></li></ol></div></li></ol></details></details></html>

brianikim commented:
https://github.com/SamZiemer/liferay-portal/pull/675

sergiojimcos committed:
LPS-154730 Add GET, DELETE and UPSERT methods by siteId and externalReferenceCode in rest-openapi

sergiojimcos committed:
LPS-154730 buildREST

sergiojimcos committed:
LPS-154730 Implement GET, DELETE and UPSERTS methods in DocumentFolderResourceImpl

sergiojimcos committed:
LPS-154730 Implement new methods to ADD, GET and FETCH folder by ERC in service modules

sergiojimcos committed:
LPS-154730 Update Folder models

sergiojimcos committed:
LPS-154730 Modify DLFolder import process

sergiojimcos committed:
LPS-154730 Update addFolder methods in other test cases from different modules

sergiojimcos committed:
LPS-154730 buildService

sergiojimcos committed:
LPS-154730 baseline

sergiojimcos committed:
LPS-154730 Update addFolder methods in other modules

Commit:
LPS-160224 Support automatic translation of web content with DeepL

sergiojimcos committed:
LPS-154730 Modify JSONFolder for externalReferenceCode field

adolfopa committed:
LPS-160224 SF

Commit:
LPS-160224 Refactor

Commit:
LPS-160224 Build Lang

noornajj committed:
LPS-169881 Add the impersonated user's id to the url in Liferay.Util.fetch

noornajj committed:
LPS-169881 Refactor all instances where doAsUserId is added to Liferay.Util.fetch

smottal committed:
COMMERCE-10419 Integration test

noornajj committed:
LPS-169881 Define the method getDoAsUserIdEncoded in Liferay.ThemeDisplay to prevent test cases from failing

smottal committed:
COMMERCE-10419 Add module dependencies

sergiojimcos committed:
LPS-173137 Delete readOnly parentTaxonomyCategory's property

sergiojimcos committed:
LPS-173137 buildREST

sergiojimcos committed:
LPS-173137 Add support to PATCH and UPDATE by taxonomyVocabularyId parameter

sergiojimcos committed:
LPS-173137 Add support to PATCH and UPDATE by parentTaxonomyCategory

ericyanLr committed:
LPS-169881 Fix additional js tests

sergiojimcos committed:
LPS-175558 Add test cases for checking nested entities in OpenAPI

sergiojimcos committed:
LPS-175558 Add nested entities in OpenAPI schema for Custom Objects and System Objects that has got a M-M and 1-M relationship

brianchandotcom committed:
LPS-154730 Wordsmith

sergiojimcos committed:
LPS-173137 Add test case for PATCH

Commit:
COMMERCE-10634 Add tests to Relevant and fix priorities

4lejandrito committed:
LPS-173137 Testing also the patch to parentAssetCategory

4lejandrito committed:
LPS-173137 SF

brianikim committed:
COMMERCE-10711 Properly support adding product options to images

brianikim committed:
COMMERCE-10711 Update methods to reflect new usage

ces-totrinh committed:
LPS-166515 Remove previous lar files

sergiojimcos committed:
LPS-154730 Delete unnessary _validateExternalReferenceCode method

CalumR23 committed:
POSHI-388 Add check to see if description message contains <>

smottal committed:
COMMERCE-10419 Semantic Versioning

smottal committed:
COMMERCE-10419 Fix linkedProduct

smottal committed:
COMMERCE-10419 Autogenerated Code (buildService)

CalumR23 committed:
POSHI-483 Add validation for multiline string in descriptions

sergiojimcos committed:
LPS-173137 Rename

lfbesada committed:
LPS-175555 Add a new method to get the different groups of a company that have articles using this URL title

Commit:
COMMERCE-10634 Adapt integers to new poshi rules

victorg1991 committed:
LPS-175839 Make the whole card draggable

victorg1991 committed:
LPS-175839 Use lexicon active styles

victorg1991 committed:
LPS-175839 Reorder toolbar buttons and add labels

victorg1991 committed:
LPS-175839 Add label

victorg1991 committed:
LPS-175839 Change card margin

Kvale1733 committed:
LRQA-78312 Disable webdriver in RemoveRedundantIndex test to save resources

allen-ziegenfus committed:
ISSITES-5990 Configure vite / clientExtension to build and render correctly

allen-ziegenfus committed:
ISSITES-5990 Update styling so that it fits within liferay - remove / replace 100vh / 100vw

allen-ziegenfus committed:
ISSITES-5990 Replace react router routing with route property

allen-ziegenfus committed:
ISSITES-5990 Output svgs to subdirectory so that we can redirect them more easily

allen-ziegenfus committed:
ISSITES-5990 Update vite dev html

ces-totrinh committed:
LPS-166515 Remove lar file when the changes have been successfully propagated from Site Template

victorg1991 committed:
LPS-175839 Scroll to the bottom of the page when adding new items

victorg1991 committed:
LPS-175839 Show delete icon only on hover

baharturk committed:
COMMERCE-10711 Add a new JSON macro

baharturk committed:
COMMERCE-10711 Add a new functional test

victorg1991 committed:
LPS-175839 Change success message when adding site navigation menu items

victorg1991 committed:
LPS-175839 Language key

victorg1991 committed:
LPS-175839 buildLang

allen-ziegenfus committed:
ISSITES-5590 Add theme extension

allen-ziegenfus committed:
ISSITES-5990 Rely on theme for clay css

allen-ziegenfus committed:
ISSITES-5990 Remove toolbar

Kvale1733 committed:
LRQA-79396 Add build-iw-jdk8 test batch

Kvale1733 committed:
LRQA-79396 Add build-lib-versions-jdk8 test batch. This will ignore publish-date changes since they can change at any time regardless of portal code changes.

Kvale1733 committed:
LRQA-79396 Add format-javadoc-jdk8 test batch

Commit:
COMMERCE-10762 - Add a verification for billing and shippingAllowed in the query search

CalumR23 committed:
POSHI-477 Add check to see if selenium method exists

CalumR23 committed:
POSHI-465 Add check for unescaped quotes in list value

victorhcunha committed:
LPS-175032 Fix METRICS_UPDATE_DUE_DATE_NO_RESULTS_MESSAGE Path

victorhcunha committed:
LPS-175032 Fix METRICS_PERFORMANCE_BY_ASSIGNEE_NO_RESULTS Path

Commit:
LPS-175919 Add more tests on open API profile

lfbesada committed:
LPS-175555 BuildService

lfbesada committed:
LPS-175555 Refactor to build a warning messages map

lfbesada committed:
LPS-175555 Adds language keys

lfbesada committed:
LPS-175555 BuildLang

lfbesada committed:
LPS-175555 Set the duplicated URL warning message when updating the article if apply

lfbesada committed:
LPS-175555 Displays the duplicated URL warning message if present

kungfumanda committed:
LPS-154743 LPS-176012 portal-search-elasticsearch7-impl: Bump elasticsearch sidecar to 7.17.9

https://issues.liferay.com/browse/LPS-176012

lfbesada committed:
LPS-175555 Displays a duplicated URL warning message when rendering the article edition view if applied.
Logic copied from UpdateArticleMVCActionCommand, only changing the language keys.

kungfumanda committed:
LPS-154743 LPS-176012 portal-search-elasticsearch7-impl: Update sidecar distribution to 7.17.9

https://issues.liferay.com/browse/LPS-176012

pat270 committed:
LPS-168532 clay:link adds iconAfter attribute to append icon at the end of link

davisaints committed:
LPS-175191 Add CanCreateActionNotificationUsingOldValueWithOnAfterUpdateInCustomObject test

pat270 committed:
LPS-168532 User Profile and Dashboard links should announce more descriptive text on screen readers

brianikim committed:
COMMERCE-10711 New ExceptionMappers to handle exceptions

vicnate5 committed:
LRQA-79396 Add javadoc reference to acceptance to remind us to enable in the future if we can

vicnate5 committed:
LRQA-79396 Add new batches to stable

allen-ziegenfus committed:
ISSITES-5990 Remove existing React app

allen-ziegenfus committed:
ISSITES-5990 Include clay css in dev index html

allen-ziegenfus committed:
ISSITES-5990 Revert vite end file lint

allen-ziegenfus committed:
ISSITES-5990 Remove public index files

haydenzheng314 committed:
LPS-175899 Remove Stream<> usage in CommerceChannelAccountEntryRelDisplayContext

ealonso committed:
LPS-176046 Removes missing occurrences

sergiojimcos committed:
LPS-175558 Fix some consistency code and add Description to the Schema from M part of a 1-M relationship

lfbesada committed:
LPS-175555 Exclude the current site and its live/staging group from validation

lfbesada committed:
LPS-175555 Reverse sort to get the default language first if present

lfbesada committed:
LPS-175555 SF Auto

marcosapmf committed:
LRAC-12719 sync custom field in the firstSync

david-truong committed:
LPS-175931 add version to title to make it less confusing when modifying in a publication

Kvale1733 committed:
LRQA-79396 auto generate, by running ant build-lib-versions

michaelhashimoto committed:
LRCI-3456 Rename client extensions from ce0*-* -> liferay-jethr0-*

shuyangzhou committed:
LPS-176311 Move expired lock clearing logic into a verify process.

jkappler committed:
LPS-176131 Integration test for Blank Site Initializer

jkappler committed:
LPS-176132 Extract to make it easier to test

jkappler committed:
LPS-176132 Adds integration test

ealonso committed:
LPS-176119 We should keep this logic removed in LPS-175547

veroglez committed:
LPS-168396 Remove header

veroglez committed:
LPS-168396 Replace liferay-ui:icon by clay:button

ccorreagg committed:
LPS-176172 Avoid overriding if isSecure is false

ealonso committed:
LPS-176119 Adds supports in ItemSelectorViewDescriptor to multiple selection

ealonso committed:
LPS-176119 When we are setting the multipleSelection to the url it is lost if we navigate through group and asset libraries selector, so, instead of setting the multipleSelection to the url, use the new method.

sergiojimcos committed:
LPS-173137 Check if the taxonomyVocabularyId from the body is null and is different from the assetCategory that we are going to update or patch

ricardocousolr committed:
LPS-122230 Keep track of all previous indexers, not just the last one, to avoid overcounting

ricardocousolr committed:
LPS-122230 Discriminate by company to avoid indexerCount to go past indexers.size()

Commit:
LPS-176173 Provide a ServletRequestWrapper to isolate the HttpSession from AuthVerifier authenticated requests

Commit:
LPS-169817 Re-invoke the AuditFilter after authenticating a request with AuthVerifierFilter

Commit:
LPS-176173 Isolate all portal session attributes unless the request is authenticated with PortalSessionAuthVerifier

ealonso committed:
LPS-176217 Extract into a new file

ealonso committed:
LPS-176217 Reorganize the logic to apply the new pattern

ccorreagg committed:
LPS-176172 Add test to validate the path and protocol is correct based on the configuration

ealonso committed:
LPS-176217 If we have more than 5 tabs use a vertical navigation instead of navigation tabs

ruben-pulido committed:
LPS-176132 Rename test class name and method name so they match ActionCommand

michaelhashimoto committed:
LRCI-3456 Rename Jethr0 classes

Commit:
LPS-176173 SemVer

claraizquierdo committed:
LPS-176170 Change allowed operators for event criteria

claraizquierdo committed:
LPS-176170 Add tests for at most operator and update tests for at least operator

hhtank committed:
LPS-166515 Deprecate class BackgroundTaskContextMapConstants in module level, add new class in portal-kernel *.constants

hhtank committed:
LPS-166515 baseline

haydenzheng314 committed:
LPS-176205 Remove Stream<> usage in CommerceShipmentFDSDataProvider

haydenzheng314 committed:
LPS-176205 Remove Stream<> usage in CommerceShipmentDisplayContext

haydenzheng314 committed:
LPS-176205 SF, inline

dantewang committed:
LPS-159031 This package is removed

ealonso committed:
LPS-176119 baseline

dantewang committed:
LPS-175899 SF, use transformToLongArray()

dantewang committed:
LPS-175899 SF, move filtering logic into transformToLongArray()

dantewang committed:
LPS-175899 SF, flip

ealonso committed:
LPS-176119 Allows define whether cet is multiple selection or not

ealonso committed:
LPS-176119 Update attending if it should be multiple selection or not

dantewang committed:
LPS-176205 SF, simplify

victorg1991 committed:
LPS-176194 Add finder for fetching the layout utility page entry by plid

victorg1991 committed:
LPS-176194 Add method in local service

victorg1991 committed:
LPS-176194 buildService

jkappler committed:
LPS-174405 Adds new language key

jkappler committed:
LPS-174405 Build lang

jkappler committed:
LPS-174405 Use correct message

brianchandotcom committed:
LPS-170904 SF

ccorreagg committed:
LPS-169912 Add validation to check if the object definition is active

ccorreagg committed:
LPS-169912 Add tests

brianchandotcom committed:
LPS-154730 Please pay this technical debt

brianchandotcom committed:
LPS-154730 Follow sql/portal-tables.sql #sfme

brianchandotcom committed:
LPS-154730 Ewe, but good enough

brianchandotcom committed:
LPS-154730 Auto SF

brianchandotcom committed:
LPS-173137 SF / Add TODO for tests

victorg1991 committed:
LPS-176194 Don't show convert to page template options for utility pages

victorg1991 committed:
LPS-176195 Change delete action for utility pages

brianchandotcom committed:
LPS-175558 SF

brianchandotcom committed:
LPS-160224 Use newer libraries

brianchandotcom committed:
LPS-160224 DeepL is their official name

brianchandotcom committed:
LPS-160224 Rename

brianchandotcom committed:
LPS-160224 Components slow things down. Let's only pay for it when it helps.

jkappler committed:
LPS-176194 Show same actions as display pages when when editing a utility page

adolfopa committed:
LPS-174913 Check for specific values, not ranges

Negative values should behave as any other invalid value instead of
silently returning an empty RSS feed.  Let the services throw the
relevant exceptions for each case.

lfbesada committed:
LPS-170531 Prevent from indexing not approved layouts

brianchandotcom committed:
LPS-160224 Why is DeepLTranslatorConfigurationModelListener#onBeforeSave checking 2 strings and this is checking 3 strings? Please resend this snippet.

brianchandotcom committed:
LPS-160224 Inline

brianchandotcom committed:
LPS-160224 Simplify

brianchandotcom committed:
LPS-160224 Simplify

allen-ziegenfus committed:
ISSITES-5990 Copy code from https://github.com/danvb112/New-paid-app-with-a-perpetual-license ec5cb1a9e070448117258a353abc0ae34cc1900e

allen-ziegenfus committed:
ISSITES-5990 blade gw formatSource

brianchandotcom committed:
LPS-160224 Simplify

allen-ziegenfus committed:
ISSITES-5990 Remove readme

allen-ziegenfus committed:
ISSITES-5990 liferay-npm-scripts format

allen-ziegenfus committed:
ISSITES-5990 liferay-npm-scripts lint:fix

brianchandotcom committed:
LPS-160224 Simplify

allen-ziegenfus committed:
ISSITES-5990 Remove react router usage

brianchandotcom committed:
LPS-160224 Simplify

brianchandotcom committed:
LPS-160224 Simplify

4lejandrito committed:
LPS-176080 Autogenerating create batch actions test

Co-authored-by: Magdalena Jedraszak <magdalena.jedraszak@liferay.com>

brianchandotcom committed:
LPS-160224 I think your logic was wrong. So I read up on https://www.deepl.com/docs-api/general/ and I believe the param is 'type'

4lejandrito committed:
LPS-176080 buildREST

Co-authored-by: Magdalena Jedraszak <magdalena.jedraszak@liferay.com>

brianchandotcom committed:
LPS-160224 We're already passing the authKey as a part?

4lejandrito committed:
LPS-176080 Overriding _getExpectedActions for reported issues pending to be resolved

    Reported issues that are still not fixed: LPS-174609, LPS-175141, LPS-175142, LPS-175138, LPS-175139, LPS-175838, LPS-175146, LPS-175145,LPS-175147, LPS-175837, LPS-175840, LPS-174610. The Form Record case has not been reported as the module will be soon deprecated.
testGetCountryRegionsPage_getExpectedActions has also been overridden as Batch Actions not yet implemented for Regions.

Co-authored-by: Magdalena Jedraszak <magdalena.jedraszak@liferay.com>

brianchandotcom committed:
LPS-160224 According the docs, one is GET and the other is POST

brianchandotcom committed:
LPS-160224 Better name

brianchandotcom committed:
LPS-160224 Better name

ccorreagg committed:
LPS-169912 Avoid returning the related elements of a inactive related Object Definition

brianchandotcom committed:
LPS-160224 Clarity

ccorreagg committed:
LPS-169912 Contribute to the OpenAPI only if the related Object Definition is active

brianchandotcom committed:
LPS-160224 Throw an exception instead

brianchandotcom committed:
LPS-160224 Simplify

claraizquierdo committed:
LPS-176170 Increse test coverage of CriteriaRowReadable component

brianchandotcom committed:
LPS-176080 SF

brianchandotcom committed:
LPS-176080 Sort

brianchandotcom committed:
LPS-176080 Wordsmith

brianchandotcom committed:
LPS-176080 SF/Wordsmith

brianchandotcom committed:
LPS-176080 prep next

brianchandotcom committed:
LPS-176080 prep next

allen-ziegenfus committed:
ISSITES-5990 Fix linting errors

allen-ziegenfus committed:
ISSITES-5990 blade gw formatSource

allen-ziegenfus committed:
ISSITES-5990 liferay-npm-scripts format

allen-ziegenfus committed:
ISSITES-5990 liferay-npm-scripts lint

nkkwon committed:
LRQA-79559 Remove commented code from test

brianchandotcom committed:
LPS-166515 Sort

allen-ziegenfus committed:
ISSITES-5990 Guard reference to buildZIPFiles (otherwise app page was not rendering)

brianchandotcom committed:
LPS-166515 Update usages

brianchandotcom committed:
LPS-166515 auto SF

david-truong committed:
LPS-176197 allow data to be keep format when rendered

shuyangzhou committed:
LPS-176311 Semantic versioning

david-truong committed:
LPS-176197 add display renderer for FragmentEntryLink

allen-ziegenfus committed:
ISSITES-5990 Remove localhost from api calls and add getProducts call

allen-ziegenfus committed:
ISSITES-5990 Call API to get app list

david-truong committed:
LPS-176197 buildLang

vicnate5 committed:
LRQA-79396 SF

CalumR23 committed:
POSHI-388 Escape characters in description element

CalumR23 committed:
POSHI-483 POSHI-388 POSHI-477 POSHI-465 Update com.liferay.poshi.core to 1.0.98

patriciajeanperez committed:
LRQA-79532 Reindex so all countries display

Commit:
LPS-175782 Update the wiki page changes with version

kenjiheigel committed:
POSHI-483 Reword

kenjiheigel committed:
POSHI-465 Use PoshiScriptParserException

austinchiang committed:
LPS-176123 Change to span

david-truong committed:
LPS-176197 baseline

gaborlovasdotcom committed:
LRQA-79597 Remove separated stable run

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
POSHI-388 Wordsmith

brianchandotcom committed:
POSHI-483 prep next

brianchandotcom committed:
LPS-169912 Wordsmith

brianchandotcom committed:
LPS-169912 Wordsmith

brianchandotcom committed:
LPS-176197 SF

brianchandotcom committed:
LPS-175555 Rename

brianchandotcom committed:
LPS-175555 Auto SF

brianchandotcom committed:
LPS-175555 Regen

Commit:
LPS-176404 Add other Data Bases to Echo Upgrade test suite

Commit:
COMMERCE-10690 Add commerce-upgrade suite

brianchandotcom committed:
LPS-175555 SF

brianchandotcom committed:
LPS-176217 SF

brianchandotcom committed:
LPS-176131 SF

brianchandotcom committed:
COMMERCE-10711 Match com.liferay.data.engine.rest.internal.jaxrs.exception.mapper.NoSuchDataLayoutExceptionMapper

brianchandotcom committed:
COMMERCE-10711 Use name from CPAttachmentFileEntryLocalServiceImpl 'String json'

ealonso committed:
LPS-176217 Rename to view_item_selector

brianchandotcom committed:
ISSITES-5990 Match sample-theme-css/client-extension.yaml

brianchandotcom committed:
ISSITES-5990 Typo

brianchandotcom committed:
ISSITES-5990 Match sample-custom-element-2/package.json

brianchandotcom committed:
ISSITES-5990 Update tools/create_remote_app.sh based on changes in this PR

brianchandotcom committed:
ISSITES-5990 Regen

brianchandotcom committed:
ISSITES-5990 Fix refresh_sample_workspace.sh

michaelhashimoto committed:
LRCI-3459 Remove unnecessary database.type from test.properties

smottal committed:
COMMERCE-10762 Fix query condition

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

brianikim commented:
https://github.com/liferay-commerce/liferay-portal/pull/3448
